operations/README.md

```diff
@@ -20,7 +20,8 @@ This section describes all operations supported by the API:
   * [Get all space blocks](enterprises.md#get-all-space-blocks) - returns space blocks in an interval.
   * [Update space state](enterprises.md#update-space-state) - updates state of a space.
   * [Add task](enterprises.md#add-task) - adds a task to the enterprise.
-  * [Add Company](enterprises.md#add-company) - adds a new company to the enterprise.
+  * [Add company](enterprises.md#add-company) - adds a new company to the enterprise.
+  * [Update company](enterprises.md#update-company) - udpates a company.
```

Comment:
typo, it should be: * [Update company](enterprises.md#update-company) - updates a company.